src/pages/staking-hub/onboarding/step5/0addNode.tsx

```diff
@@ -70,10 +71,18 @@ export default function AddNode() {
             });
           }
         });
-        set_isLoading(false);
+      set_isLoading(false);
     }
   };
 
+
+  const addressIsOwnerAddress = () => {
+    if (ownerAddress === address) {
+      return true
+    }
+    return false
+  }
```

Comment:
Sometimes the addresses can have different size of cases

```suggestion
   return ownerAddress?.toLocaleLowerCase() === address?.toLocaleLowerCase()
```